geemusic/utils/music.py

```diff
@@ -23,6 +23,14 @@ def __init__(self, username, password, logger=None):
         )
         self.indexing_thread.start()
 
+    def populate_library(self): #TODO: Use this as a function to refresh the library with Alexa via voice commands.
```

Comment:
is this function used anywhere? if not let's remove it.

Reply:
nope. deleted it